app/views/categories/index.html.erb

```diff
@@ -0,0 +1,92 @@
+<div class="header">
+  <%= link_to "<i class='fas fa-arrow-left'></i>".html_safe, :back, class: "btn" %>
+  <h1>Categories</h1>
+  <%= form_for(current_user, url: destroy_user_session_path, method: :delete, html: { class: "nav-link" }) do |f| %>
+    <%= f.button type: :submit, class: "btn-logout" do %>
+      <%= content_tag :i, "", class: "fa-solid fa-right-from-bracket" %>
+    <% end %>
+  <% end %>
+</div>
+<% if @categories.any? %>
+  <ul class="categories">
+    <% @categories.each do |category| %>
+      <li class="category">
+        <% if category.icon.present? %>
+          <%= raw category.icon %>
+        <% end %>
+        <h3><%= category.name %>:</h3>
+        <p>Total: $<%= category.expenses.collect(&:amount).sum %></p>
+        <div class="btn">
+          <button><a href="<%= category_expenses_path(category) %>">View Expenses</a></button>
+          <button><%= link_to 'Add a New Expense', new_category_expense_path(category) %></button>
+        </div>
+      </li>
+    <% end %>
+  </ul>
+  <button class="btn"><%= link_to 'Add a New Category', new_category_path %></button>
+<% else %>
+  <p class="msg">No categories found. Click the button below to add a new category.</p>
+  <button class="btn"><%= link_to 'Add a New Category', new_category_path%></button>
+<% end %>
+
+<style>
```

Comment:
- [ ] I would encourage you to stick to the design guidelines as much as possible including Colors, Typography(font face, size, and weight), and Layout (composition and space between elements). 

> Design Guide
<img width="319" alt="Screenshot 2023-05-19 at 23 43 07" src="https://github.com/Salwa99/My-Budget/assets/71140133/1fd37099-c6bb-4a25-850a-9043619defff">

> Your design
<img width="374" alt="Screenshot 2023-05-19 at 23 48 34" src="https://github.com/Salwa99/My-Budget/assets/71140133/73dd39f5-89b6-4842-a23a-bda9c225e171">

Reply:
I'm gonna fix this minor issue even though the  main purpose of the capstone is to build an application that respects the logic of the project, and has nearly the same as the original design